src/components/GlobalLinterMessages/index.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import LinterMessage from '../LinterMessage';
+import {
+  fakeExternalLinterMessage,
+  createFakeLocation,
+  createContextWithFakeRouter,
+} from '../../test-helpers';
+import { createInternalMessage } from '../../reducers/linter';
+import { getCodeLineAnchorID } from '../CodeView/utils';
+
+import GlobalLinterMessages, { PublicProps } from '.';
+
+describe(__filename, () => {
+  const render = (props: PublicProps) => {
+    return shallow(<GlobalLinterMessages {...props} />);
+  };
+
+  const renderWithMount = (props: PublicProps) => {
+    const location = createFakeLocation();
+    const context = createContextWithFakeRouter({ location });
+    return mount(<GlobalLinterMessages {...props} />, context);
+  };
+
+  it('renders no linter messages', () => {
+    const root = render({ messages: [] });
+    const div = root.find('div');
+
+    expect(div).toHaveLength(0);
```

Comment:
If the top-level wrapper changes from a `div` to another component (for example), this test shouldn't fail. Here's a more robust way to make the same assertion:

```diff
diff --git a/src/components/GlobalLinterMessages/index.spec.tsx b/src/components/GlobalLinterMessages/index.spec.tsx
index 098eb24..74215f6 100644
--- a/src/components/GlobalLinterMessages/index.spec.tsx
+++ b/src/components/GlobalLinterMessages/index.spec.tsx
@@ -25,9 +25,8 @@ describe(__filename, () => {
 
   it('renders no linter messages', () => {
     const root = render({ messages: [] });
-    const div = root.find('div');
 
-    expect(div).toHaveLength(0);
+    expect(root.type()).toEqual(null);
   });
 
   it('renders multiple linter messages', () => {

```